rmgpy/molecule/adjlist.py

```diff
@@ -381,7 +367,6 @@ def fromOldAdjacencyList(adjlist, group=False, saturateH=False):
         if not group:
             if saturateH:
                 # Add explicit hydrogen atoms to complete structure if desired
-                orders = {'S': 1, 'D': 2, 'T': 3, 'B': 1.5}
                 standardLonePairs = {'H': 0, 'C': 0, 'O': 2, 'S': 2, 'Si': 0, 'Cl': 3, 'He': 1, 'Ne': 4, 'Ar': 4}
```

Comment:
After looking at this a bit more, I think the two dictionaries should be the same, and nitrogen should be included. Otherwise, there will be a KeyError in line 380 if this function is ever called with a nitrogen atom. I suppose this has never come up because `fromOldAdjacencyList` doesn't get used very much?